src/main/java/org/kainos/ea/db/jobDao.java

```diff
@@ -0,0 +1,31 @@
+package org.kainos.ea.db;
+import org.kainos.ea.cli.JobRequest;
+
+import java.sql.*;
```

Comment:
IntelliJ will have made this change, but can you make this not a wildcard import